core/arch/proc.h

```diff
@@ -294,14 +294,15 @@ extern size_t cache_line_size;
 
 #define CACHE_LINE_SIZE() cache_line_size
 
-/* xcr0 and xstate_bv feature bits */
+/* xcr0 and xstate_bv feature bits, as actually used by the processor. */
 enum {
-    XCR0_HI16_ZMM = 7,
-    XCR0_ZMM_HI256 = 6,
-    XCR0_OPMASK = 5,
-    XCR0_AVX = 4,
-    XCR0_SSE = 2,
-    XCR0_FP = 1,
+    XCR0_HI16_ZMM = 0x80,
+    XCR0_ZMM_HI256 = 0x40,
```

Comment:
These two similarly-named flags are confusing to those who haven't looked at the manual.  Please add comments clarifying which one is which (the upper part of the zmm0-15 registers vs the entire part of the new registers zmm16-31).

Reply:
Done.